//> error "continue stmt must be inside a loop"

import std/core;

func main() {
	continue;
}
